backend/habitat/viewsets.py

```diff
@@ -490,3 +510,54 @@ def zones_iucn(request):
                     break
     print(zones_iucn)
     return Response(zones_iucn)
+
+@action(detail=False)
+@api_view()
+def habitat_statistics(request):
+    network   = request.query_params.get('network')
+    park      = request.query_params.get('park')
+    zone      = request.query_params.get('zone')
+    zone_iucn = request.query_params.get('zone-iucn')
+
+    habitat_stats = []
+    total_area = 0
+
+    with connections['transects'].cursor() as cursor:
+        if (park):
```

Comment:
why the parentheses here?  (I don't think they do anything)

Reply:
Right you are! Removing them.